plotgui.py

```diff
@@ -351,7 +469,8 @@ def __init__(self, model, FM, parent=None):
 
         # Create submit button
         self.applyButton = QPushButton("Apply Changes")
-        self.applyButton.setMinimumHeight(self.FM.height() * 1.6) # Mac bug fix
+        # Mac bug fix
```

Comment:
Should this only be present for `sys.platform == 'darwin'`?

Reply:
I'm honestly not sure. This was pre-existing. I can look into it though!

Reply:
I've built the plotter on OSX now and there are some other issues I'll need to address as well. Mind if I wrap all of those into a future PR together?

Reply:
Sounds good